deployments/apiv2/services/users/src/main.py

```diff
@@ -804,7 +804,7 @@ async def get_all_users(request: Request, token=Depends(ProtectedAny(tag=ScopeTa
     query = (
         "SELECT u.id, u.name, u.email, u.created_at, u.last_login, u.verified, u.suspended, u.reset_token, array_agg(s.scope) as scopes "
         "FROM users u "
-        "INNER JOIN account_scopes s ON u.id=s.user_id "
+        "LEFT JOIN account_scopes s ON u.id=s.user_id "
```

Comment:
Switching to left join so that if for some reason a user (or customer below, see similar query) has no scopes in account_scopes, it still returns the user/customer entry with scopes as [] and doesn't just omit it